weareinreach/inreach-catalog
# include a Tooltip and Information icon for Mail Order Services

## Description
added an information icon and tooltip with clickable text to the Mail Order Service subcategory
made the Tooltip show up on click for mobile devices - for Mail Order Service subcategory
also fixed up the tooltip for vetted information in the DetailHeader mobile view


## Asana ticket:
https://app.asana.com/0/1132189118126148/1202515609917967

## PR Checklist




## How to Test

- Open Service Type Filter
- Hover/Select Abortion Care
- Sub category list should open
- Mail Order services should have an information icon
- For Desktop Browsers - hover over the Mail Order Services element - a Tooltip with clickable text should appear
- click on the text - a link should open in another tab
- For mobile devices - click the element to show the tooltip - a Tooltip with clickable text should appear
- click on the text - a link should open in another tab
-
